docs/GettingStartedDocs/Contributors/building_oe_sdk.md

```diff
@@ -33,22 +33,25 @@ Please refer to the following [documentation](/docs/GettingStartedDocs/SGXSuppor
 
    On Linux, if your target system does not have any SGX hardware support, you may want to choose simulation.
 
-   On Windows, Open Enclave SDK does not support simulation mode.
+   On Windows, Open Enclave SDK supports simulation mode but only on systems with SGX1 or SGX1+FLC support.
 
 #### 3. Build, install and run
 
    Choose an operating mode that is compatible with the SGX support level of your target system.
    The links below contain instructions on how to set up Open Enclave SDK environment for a given mode.
 
-On Linux:
-  - [Setup Open Enclave SDK for SGX1+FLC](SGX1FLCGettingStarted.md)
-  - [Setup Open Enclave SDK for SGX1](SGX1GettingStarted.md)
-  - [Setup Open Enclave SDK for simulation mode](SimulatorGettingStarted.md)
+On Ubuntu 1604 or Ubuntu 1804
+  - [Set up Open Enclave SDK for SGX1+FLC](SGX1FLCGettingStarted.md)
+  - [Set up Open Enclave SDK for SGX1](SGX1GettingStarted.md)
+  - [Set up Open Enclave SDK for simulation mode](SimulatorGettingStarted.md)
 
-On Windows:
+On RHEL8
+ - [Set up of Open Enclave SDK on RHEL8](ExperimentalSupportRHEL8.md)
+
+On Windows
  - [Set up Open Enclave SDK for SGX1+FLC](WindowsSGX1FLCGettingStarted.md)
  - [Set up Open Enclave SDK for SGX1](WindowsSGX1GettingStarted.md)
- - Simulation mode is not supported on Windows
+ - Simulation mode is supported on Windows but only on systems that have SGX1 or SGX1+FLC support.
```

Comment:
Isn't that the case too on Linux? Have we tested whether we can build OE SDK on an AMD x64 machine and run simulation mode?

Reply:
Yes. Have verified that we can build Oe sdk on non SGX vm and run tests in simulation mode

Reply:
It is mysterious how we are able to successfully build without the SGX EnclaveCommon API Library.

Have we tried using the SDK on non-sgx vm as an user as opposed to building it as a contributor?

Reply:
@anakrish I believe there to be confusion a foot. 

One can install the SGX EnclaveCommon API on a non-sgx LINUX vm. One can also do some hacky things to build the OE SDK on a non-sgx Windows VM. Simulation on non-sgx windows, I am not sure.

For example, if you run through the getting started docs for SGX/FLX on a vanilla ubuntu VM, building is successful and simulation mode can be run

Reply:
On Ubuntu you have to install libsgx-enclave-common. I have not tried on an amd machine. Maybe I say on an intel non SGX machine.

Reply:
ansible-playbook scripts/ansible/oe-contributors-setup.yml sets up libsgx-enclave-common as well.